software/docker/kafka-can-log/kafka_can_log/Makefile.in

```diff
@@ -0,0 +1,562 @@
+# Makefile.in generated by automake 1.15.1 from Makefile.am.
```

Comment:
I don't think this is supposed to be committed. We should add/update `.gitignore`